chats/views.py

```diff
@@ -3,13 +3,20 @@
 # Create your views here.
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-from .models import *
+from .models import ChatGroup
 from .forms import ChatmessageCreateForm
 
+def chat_view_home(request):
+    chat_groups = ChatGroup.objects.all()
+    return render(request, "chats/chat_home.html", {"chat_groups": chat_groups})
 
-@login_required
-def chat_view(request):
-    chat_group = get_object_or_404(ChatGroup, group_name="public-chat")
+def chat_view_new(request):
```

Comment:
```suggestion
def chat_view_new(request):
```
為求簡潔，可以命名為 `chat_new` 就好